WebHostLib/options.py

```diff
@@ -84,7 +84,7 @@ def get_html_doc(option_type: type(Options.Option)) -> str:
                     "max": option.range_end,
                 }
 
-                if issubclass(option, Options.SpecialRange):
+                if issubclass(option, Options.NamedRange):
                     game_options[option_name]["type"] = 'special_range'
```

Comment:
Shouldn't the javascript-side name also change, to stay in sync?

Reply:
This will include a few other places where you need to change it. Take a look at the places I had to change in #2305

Reply:
Yes, but I am suggesting changing the type here, for consistency, and subsequently also adapting the js files to match

Reply:
I agree that it should be matched on the JS side for consistency. Searching for the term across the project should find its variants, it's nice.